### PR TITLE
Add Conflux eSpace

### PR DESCRIPTION
**Pull request by darwintree**
## Description

Add Conflux eSpace

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

**Review by tmm — CHANGES_REQUESTED** [review]
Looking good. The [multicall3 contract](https://evm.confluxscan.io/address/0xeff0078910f638cd81996cc117bccd3edf2b072f?tab=contract-viewer) isn't verified though. Any chance you can verify or deploy another that is?

**Comment by darwintree** [comment]
> Looking good. The [multicall3 contract](https://evm.confluxscan.io/address/0xeff0078910f638cd81996cc117bccd3edf2b072f?tab=contract-viewer) isn't verified though. Any chance you can verify or deploy another that is?

Sorry for that. 
Previous verified at another domain(https://evm.confluxscan.net/address/0xeff0078910f638cd81996cc117bccd3edf2b072f) but the verification data is somehow not shared. 
Now both are verified(https://evm.confluxscan.io/address/0xeff0078910f638cd81996cc117bccd3edf2b072f?tab=contract-viewer)
BTW: the 2 scan provided at different domain is generally exactly the same but with different server location(`.net` at Hong Kong and `.io` at US East) and it is the same for public RPCs(https://doc.confluxnetwork.org/docs/espace/build/network-endpoints). Although current configuration is generally ok, but I wonder that if I want to add `.net`  ones, what is the recommended way to label these 2 domains?

File: packages/chains/src/cfxESpace.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Chain } from './types'
+
+export const cfxESpace = {
```
Comment (tmm) [inline packages/chains/src/cfxESpace.ts:3]: Should we call this `confluxESpace` instead? Bit more verbose, but is more easily searchable and autocomplete-able.

Comment (darwintree) [reply packages/chains/src/cfxESpace.ts]: Renamed

**Review by tmm — APPROVED** [review]

**Pull request merged**